api/diff.go

```diff
@@ -30,77 +30,72 @@ func apiDiffHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-type diffResult struct {
-	Diff    shared.ResultsDiff `json:"diff"`
-	Renames map[string]string  `json:"renames,omitempty"`
+func loadDiffRuns(store shared.Datastore, q url.Values) (shared.TestRuns, error) {
+	if runIDs, err := shared.ParseRunIDsParam(q); err != nil {
+		return nil, err
+	} else if len(runIDs) > 0 {
```

Comment:
Not particularly fond of the fact that `ParseRunIDsParam` could return `nil, nil` instead of `[], nil`, and `len(nil)` would throw an error here. See [here](https://github.com/web-platform-tests/wpt.fyi/blob/76e6b108d32efba7d38ae96e1bf0334b25c64e2a/shared/params.go#L475)

Reply:
Thanks for checking this! This won't be an error because the `nil` here is typed: https://play.golang.org/p/EECu3SNLH4q